client-internal/src/main/java/org/terracotta/angela/client/support/junit/AngelaRule.java

```diff
@@ -94,7 +98,12 @@ protected void before(Description description) throws Throwable {
       }
     }
 
-    this.clusterFactory = new ClusterFactory(description.getTestClass().getSimpleName(), configuration);
+    String id = description.getTestClass().getSimpleName();
+    if (description.getMethodName() != null) {
+      id += "." + description.getMethodName();
+    }
```

Comment:
Tested on Win ? Not too long ?

Reply:
Why not other have 2 folders: `<test-class>/<test-name>`.
it would be better I think considering the number of tests we have,